python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -46,17 +81,16 @@ def convert_to_cudf(cp_arrays, weight_t):
 
 
 def _call_plc_uniform_neighbor_sample(
-    sID, mg_graph_x, st_x, fanout_vals, with_replacement
+    sID, mg_graph_x, st_x, fanout_vals, with_replacement, weight_t
 ):
-    return pylibcugraph_uniform_neighbor_sample(
+    cp_arrays = pylibcugraph_uniform_neighbor_sample(
         resource_handle=ResourceHandle(Comms.get_handle(sID).getHandle()),
         input_graph=mg_graph_x,
         start_list=st_x,
         h_fan_out=fanout_vals,
         with_replacement=with_replacement,
```

Comment:
You are missing a required positional [argument](https://github.com/rapidsai/cugraph/blob/branch-22.12/python/pylibcugraph/pylibcugraph/uniform_neighbor_sample.pyx#L70) after `with_replacement` : `do_expensive_check=False`. Without that, you will get an error. Sorry if I wasn't clear when I meant to remove the expensive check. I meant to set it to `False`